Dockerfile

```diff
@@ -1,8 +1,23 @@
-FROM alpine:3.18
+FROM debian:latest
```

Comment:
Can we pin this to a specific Debian version? If feasible, it might be better to use an Alpine image to reduce the overall image size. That'll save Exercism money since we're deploying 71+ different Docker images to each test runner machine in the cloud.

Reply:
I can definitely pin this to debian:bookworm instead. However, I tried using alpine, but Roc failed, probably because I need to install some libraries. I'll start with debian:bookworm, and I'll switch to alpine asap.

Reply:
If Alpine doesn't work out, that's fine. It's just a nice starting point for reducing file size, but I don't believe all tracks can use Alpine images due to some limitations in what's available.

Reply:
Trying to solve the `DlOpen` error (see above), I've switched to `Ubuntu 22.24`. Still works fine on my machine but fails in Github CI. I think it might be a permission issue on the `/tmp` folder (see this [SO answer](https://stackoverflow.com/a/55693490/38626)).

Reply:
What I know about Docker or Github CI can fit in a thimble, but Erik can probably help you out further. He's pretty handy. :)

Reply:
I looked into migrating to Alpine, but Roc is currently built on glibc whereas Alpine uses musl. There's an [ongoing effort to make Roc support musl](https://github.com/roc-lang/roc/issues/6058), so we should be able to migrate to Alpine once that work is completed.